frontend/src/tests/components/eMIB/EditActionDialog.test.js

```diff
@@ -210,3 +210,29 @@ function testMode(actionType, editMode) {
     );
   }
 }
+
+it("clicking on the button only adds the email once", () => {
+  const addEmail = jest.fn();
+  const handleClose = jest.fn();
+  const wrapper = mount(
+    <EditActionDialog
+      emailId={1}
+      emailSubject={"hello team"}
+      showDialog={true}
+      handleClose={handleClose}
+      addEmail={addEmail}
+      addTask={() => {}}
+      updateEmail={() => {}}
+      updateTask={() => {}}
+      readEmail={() => {}}
+      actionType={ACTION_TYPE.email}
+      editMode={EDIT_MODE.create}
+    />
+  );
+  wrapper.find("#unit-test-email-response-button").simulate("click");
+  // In the test, calling handleClose does not change the showDialog vale
```

Comment:
You just have a little typo here: _vale_ instead of _value_

Reply:
Fixed in 639cf20